cluster/src/main/scala/org/apache/spark/executor/SnappyCoarseGrainedExecutorBackend.scala

```diff
@@ -92,7 +92,7 @@ class SnappyCoarseGrainedExecutorBackend(
     if (executor != null) {
       // kill all the running tasks
       // InterruptThread is set as true.
```

Comment:
can you change this comment and also add the explanation as in the PR description?